pinot-integration-tests/src/test/java/org/apache/pinot/integration/tests/ControllerPeriodicTasksIntegrationTests.java

```diff
@@ -472,7 +474,26 @@ public void afterBrokerResourceValidationManagerTest(ITestContext context)
     dropOfflineTable(table2);
   }
 
-  // TODO: tests for other ControllerPeriodicTasks (RetentionManager, OfflineSegmentIntervalChecker, RealtimeSegmentValidationManager)
+  @Test
```

Comment:
this new test is not conforming to the sequential tests nature described in that comment. I think it is fine for your test, as you are not creating or changing any states. But it would be good to mention that in the comment so as to not confuse anyone

Reply:
added